packages/protons-benchmark/buf.gen.yaml

```diff
@@ -0,0 +1,6 @@
+version: v1
```

Comment:
Is there a way for this config file to live in the `protobuf-es` folder?  It'd be more manageable not to have implementation-specific files outside of their respective directories.